packages/unminify/src/transformations/un-es6-class.ts

```diff
@@ -1,5 +1,6 @@
 import wrap from '../wrapAstTransformation'
 import type { ASTTransformation } from '../wrapAstTransformation'
+import type { ExpressionKind } from 'ast-types/lib/gen/kinds'
 import type { AssignmentExpression, CallExpression, ExpressionStatement, FunctionExpression, Identifier, MemberExpression, VariableDeclarator } from 'jscodeshift'
```

Comment:
I haven't checked to see if `jscodeshift` re-exports this, but curious if it could have been imported from there like the other AST types seem to have been?

Reply:
All *kind type aren't re-exported by `jscodeshift`.

Reply:
Cool cool, makes sense. Thanks for the knowledge :)